aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ScheduledV2Event.java

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazonaws.services.lambda.runtime.events;
+
+import org.joda.time.DateTime;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * represents a scheduled V2 event
+ */
+public class ScheduledV2Event implements Serializable, Cloneable {
```

Comment:
I think we tend to favour Lombok annotations atm, instead of getters, setters, toString etc..

```
@Data
@Builder(setterPrefix = "with")
@NoArgsConstructor
@AllArgsConstructor
```